addons/intel/e810.go

```diff
@@ -50,6 +50,14 @@ done
 echo "Disabled all SMA and U.FL Connections"
 `
 
+func getDefaultUblxCmds() []E810UblxCmds {
+	defaultUblxCommands := E810UblxCmds{
+		ReportOutput: false,
+		Args:         []string{"-p", "CFG-MSG,1,34,1"},
```

Comment:
add comments:
Set the receiver to output the message NAV-CLOCK once every nav solution.